client/src/browse/filterBar/FilterBar.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+
+const FilterBar = (props) => {
```

Comment:
@MarkAlperin I would recommend destructuring props to reduce property lookups and enhance readability (imo).

"const FilterBar = (props) => {"    

becomes

"const FilterBar = ({ itemData, setFilteredItems,  setSelectedItem }) =>  { "

(I think I got all of the props)

and then remove "props." below where ever it is accessed.

Reply:
can I consider this tech-debt for now and return when I have time/non-broken components?